### PR TITLE
JSON update

### PR DESCRIPTION
**Pull request by grzanka**
New file from UI example and tests update.
Previous JSON samples were from 0.2 version, now we have 0.6
Moreover I've removed hardcoded parts of JSON from tests, now they are replaced by "expected results file"

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -264,7 +264,3 @@ cython_debug/
 .ionide
 
 # End of https://www.toptal.com/developers/gitignore/api/python,visualstudiocode,pycharm
-beam.dat
-detect.dat
-geo.dat
-mat.dat
```
Comment (hendzeld) [inline .gitignore:270]: Did these lines serve any purpose? Shouldn't we have a similar rule?

Maybe it should be forbidden to commit *.dat files except in some locations?
Something like:

```.gitignore
**/*.dat
!input_examples/expected_shieldhit_output/*.dat
```

or alternatively, we can globally ignore all `*.dat` files 
```.gitignore
**/*.dat
```
and add an additional `.gitignore` file in `input_examples/expected_shieldhit_output`
```.gitignore
!*.dat
```

File: tests/test_shieldhit_converter.py [hunk 2]
```diff
@@ -86,40 +18,27 @@ def parser() -> Parser:
...
+@pytest.mark.parametrize('filename', ['geo.dat'])
+@pytest.mark.skip(reason="Something is wrong with geo.dat file.")
+def test_to_be_fixed(parser: Parser, default_json : dict, tmp_path : Path, filename: str) -> None:
+    """Check if all output files are created"""
+    logging.info('Checking %s file', filename)
+    run_parser(parser, default_json, tmp_path)
+    dir_with_expected_files = Path(__file__).parent.parent / 'input_examples' / 'expected_shieldhit_output'
+    assert (tmp_path / filename).exists() is True
+    assert (dir_with_expected_files / filename).exists() is True
+    assert filecmp.cmp(tmp_path / filename, dir_with_expected_files / filename) is True
```
Comment (hendzeld) [inline tests/test_shieldhit_converter.py:44]: Maybe pedantic but github complains about a missing newline character.

File: tests/test_shieldhit_converter.py [hunk 3]
```diff
@@ -86,40 +18,27 @@ def parser() -> Parser:
...
-        assert f.read() == _Geo_str
+@pytest.mark.parametrize('filename', ['beam.dat', 'mat.dat', 'detect.dat'])
+def test_if_expected_files_created(parser: Parser, default_json : dict, tmp_path : Path, filename: str) -> None:
+    """Check if all output files are created"""
+    logging.info('Checking %s file', filename)
+    run_parser(parser, default_json, tmp_path)
+    dir_with_expected_files = Path(__file__).parent.parent / 'input_examples' / 'expected_shieldhit_output'
+    assert (tmp_path / filename).exists() is True
+    assert (dir_with_expected_files / filename).exists() is True
+    assert filecmp.cmp(tmp_path / filename, dir_with_expected_files / filename) is True
```
Comment (hendzeld) [inline tests/test_shieldhit_converter.py:33]: Shouldn't it be: `assert X` instead `assert X is True`?

File: tests/test_shieldhit_converter.py (hunk 2 above)
Comment (hendzeld) [inline tests/test_shieldhit_converter.py:44]: Shouldn't it be: `assert X` instead `assert X is True`?

**Review by hendzeld — COMMENTED** [review]
My suggestions

File: tests/test_shieldhit_converter.py (hunk 2 above)
Comment (grzanka) [reply tests/test_shieldhit_converter.py]: Done in 089ad72ab7919d792fcfbef864cf85543abf2363

File: tests/test_shieldhit_converter.py (hunk 3 above)
Comment (grzanka) [reply tests/test_shieldhit_converter.py]: Done in 089ad72ab7919d792fcfbef864cf85543abf2363

File: .gitignore (hunk 1 above)
Comment (grzanka) [reply .gitignore:270]: Done in 089ad72ab7919d792fcfbef864cf85543abf2363

File: tests/test_shieldhit_converter.py (hunk 2 above)
Comment (grzanka) [reply tests/test_shieldhit_converter.py]: Done in a833e93e05c6f4c52124062c9d83c2b7a3646a05

File: tests/test_shieldhit_converter.py [hunk 4]
```diff
@@ -1,125 +1,44 @@
...
-    output_dir = tmp_path_factory.mktemp(_Test_dir)
-    return output_dir
+from converter.api import get_parser_from_str, run_parser
+from converter.shieldhit.parser import Parser
 
 
 @pytest.fixture
 def parser() -> Parser:
-    """Just a praser fixture."""
+    """A parser for SHIELD-HIT12A fixture."""
```
Comment (hendzeld) [inline tests/test_shieldhit_converter.py:14]: Minor suggestion:

For me, it sounds like this `function` is making a parser of SHIELD-HIT12A fixture.

Wouldn't `A fixture of SHIELD-HIT12A parser` be a better description?
Alternatively `SHIELD-HIT12A parser fixture`

**Review by hendzeld — COMMENTED** [review]
Suggestion

File: tests/test_shieldhit_converter.py (hunk 4 above)
Comment (grzanka) [inline tests/test_shieldhit_converter.py:14]:
```suggestion
    """SHIELD-HIT12A parser fixture"""
```

Comment (grzanka) [reply tests/test_shieldhit_converter.py]: Done

**Review by hendzeld — APPROVED** [review]
LGTM

**Pull request merged**